### PR TITLE
Refs #25217 - Refactor hosts/vmware/disk component

### PR DESCRIPTION
**Pull request by sharvit**
Adds prop-types, default-props and fix some code styling

### CODE REVIEW

File: webpack/assets/javascripts/react_app/components/hosts/storage/vmware/controller/disk/index.js [hunk 1]
```diff
@@ -30,50 +28,48 @@ const Disk = ({
...
+      <label className="col-md-2 control-label">{__('Disk name')}</label>
+      <div className="col-md-4">{name}</div>
       <div className="col-md-2">
-        {!vmExists &&
-        <Button className="close" onClick={removeDisk}>
-          <span aria-hidden="true">×</span>
-        </Button>}
+        {!vmExists && (
+          <Button className="close" onClick={removeDisk}>
+            <span aria-hidden="true">×</span>
```
Comment (ohadlevy) [inline webpack/assets/javascripts/react_app/components/hosts/storage/vmware/controller/disk/index.js]: nitpick: shoudl this be ```&times```

Comment (sharvit) [reply webpack/assets/javascripts/react_app/components/hosts/storage/vmware/controller/disk/index.js]: Sorry @ohadlevy I'm not sure I understand, can you explain please?

**Comment by ohadlevy** [comment]
replace ```x``` with ```&times;```

**Comment by sharvit** [comment]
Done. Thanks @ohadlevy

**Review by ohadlevy — APPROVED** [review]

**Review by chris1984 — APPROVED** [review]

**Review by ohadlevy — APPROVED** [review]

**Pull request merged**